proposals/filecoin-retrieval-in-ipfs.md

```diff
@@ -0,0 +1,163 @@
+# Filecoin Retrieval In IPFS
+
+Authors: @hannahhoward
+
+Initial PR: TBD <!-- Reference the PR first proposing this document. Oooh, self-reference! -->
+
+Problems Addressed:
+
+- Data that has been stored successfully is not guaranteed to be retrievable. (P1)
```

Comment:
In terms of how the proposal solves this problem, is the assumption that lotus nodes being able to add DHT records and IPFS to retrieve from lotus nodes will improve retrieval guarantees because there will be more demand for retrieval (so more reason for lotus nodes to supply the data)?